support-frontend/assets/pages/showcase/components/ctaSubscribe.jsx

```diff
@@ -26,7 +26,7 @@ export default function CtaSubscribe() {
         <AnchorButton
           icon={<ArrowRightStraight />}
           href="/subscribe"
-          onClick={() => sendClickedEvent('support_page_cta_subscribe')()}
```

Comment:
as discussed it does seem a bit confusing that the old one had the extra set of `()` wheras the new one doesn't.  Although the new function does look like things are more sensible!  As discussed it might always be worth checking quickly.